src/ray/raylet/node_manager.cc

```diff
@@ -1842,6 +1842,38 @@ void NodeManager::HandleRequestWorkerLease(const rpc::RequestWorkerLeaseRequest
   SubmitTask(task, Lineage());
 }
 
+void NodeManager::HandleRequestResourceReserve(
```

Comment:
When we implement placement groups for the new scheduler, we'll be able to add unit tests properly for that.